vm/src/vm/runners/cairo_pie.rs

```diff
@@ -72,10 +72,11 @@ pub struct OutputBuiltinAdditionalData {
     pub attributes: Attributes,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq)]
 #[serde(untagged)]
 pub enum BuiltinAdditionalData {
     // Contains verified addresses as contiguous index, value pairs
+    Empty([usize; 0]),
```

Comment:
Why not just use an empty variant here?

Reply:
I need this variant to deserialize a JSON empty list, changing `usize` to the empty type.